src/code-examples/tabs/css.js

```diff
@@ -1,40 +1,39 @@
-const css = `
-    .is-hidden {
-      display: none;
-    }
-    
-    /* ================ */
-    /* = EXTRA STYLES = */
-    /* ================ */
-    
-    [role="tab"] {
-      border: none;
-      padding: 0.35rem;
-      font-size: 1.4rem;
-      color: black;
-      cursor: pointer;
-      transition: all ease-in-out 100ms;
-      border: 2px solid white;
-      border-bottom: none;
-    }
-    
-    [role="tab"]:focus,
-    [role="tab"]:hover {
-      border: 2px solid black;
-      border-bottom: none;
-    }
-    
-    [aria-selected="true"] {
-      border: 2px solid black;
-      border-bottom: none;
-    }
-    
-    [role="tabpanel"] {
-      color: black;
-      padding: 0 1rem 1rem 1rem;
-      max-width: 30rem;
-      border: 1px solid black;
-    }
-  `;
+const css = `.is-hidden {
+  display: none;
+}
+
+/* ================ */
+/* = EXTRA STYLES = */
+/* ================ */
+
+[role="tab"] {
+  border: none;
+  padding: 0.35rem;
+  font-size: 1.4rem;
+  color: black;
+  cursor: pointer;
+  transition: all ease-in-out 100ms;
+  border: 2px solid white;
+  border-bottom: none;
+}
+
+[role="tab"]:focus,
+[role="tab"]:hover {
+  border: 2px solid black;
+  border-bottom: none;
+}
+
+[aria-selected="true"] {
+  border: 2px solid black;
+  border-bottom: none;
+}
+
+[role="tabpanel"] {
+  color: black;
+  padding: 0 1rem 1rem 1rem;
+  max-width: 30rem;
+  border: 1px solid black;
+}
```

Comment:
This is weird for sure. The only reason I changed these to make them look WORSE in our files, is so that in the Sandpack editor, they look normal, with the proper indentations. It makes our code really gross but the Sandpack look nice. 
<img width="326" alt="Screen Shot 2022-12-13 at 5 11 44 PM" src="https://user-images.githubusercontent.com/69602589/207455713-a1e81fe6-cac2-47e0-aec6-6b766c63a7db.png">

Reply:
Oh that makes sense, good job